drivers/include/sx126x.h

```diff
@@ -42,19 +56,22 @@ typedef struct {
     gpio_t busy_pin;                    /**< Busy pin */
     gpio_t dio1_pin;                    /**< Dio1 pin */
     sx126x_reg_mod_t regulator;         /**< Power regulator mode */
+    void(*set_rf_mode)(sx126x_t *dev, sx126x_rf_mode_t rf_mode); /**< Interface
+    to set RF switch parameters */
```

Comment:
This doesn't look that nice,  I think I would prefer it on one line or as a brief or something.

Reply:
Done